charon/config.py

```diff
@@ -102,8 +124,10 @@ def __init__(self, data: Dict):
         self.__ignore_signature_suffix: Dict = data.get("ignore_signature_suffix", None)
         self.__signature_command: str = data.get("detach_signature_command", None)
         self.__aws_cf_enable: bool = data.get("aws_cf_enable", False)
+        self.__radas_config_enable: bool = data.get("radas_config_enable", False)
```

Comment:
After thinking, we don't need an extra "enable" here, instead we can use "radas_config and radas_config.validate" to decide if radas is enabled.

Reply:
@ligangty sounds good, already update in the new commit, BTW I've committed code in the different commits with different aspects mentioned in the review, which might be more convenient to check.